### ISSUE TITLE
parallel handling of notifications

### ISSUE THREAD

**Issue by adlerhurst** [zitadel/zitadel#8932]
### Preflight Checklist

- [x] I could not find a solution in the existing issues, docs, nor discussions
- [x] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Describe your problem

Notifications currently are sent out sequentially per instance. This can lead to incremental delivery times of notifications if providers take long to acknowledge the send request.

### Describe your ideal solution

As a system administrator I would expect that multiple messages can be sent out in parallel so that the delivery times do not pile up.

### Version

v2.65

### Environment

ZITADEL Cloud

**Comment by adlerhurst** [comment]
duplicate of https://github.com/zitadel/zitadel/issues/8931
